Docs: Improve `SHOW` syntax page (#75470)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/show.md

```

`SHOW TABLE t` and `SHOW DATABASE db` have the same meaning as `SHOW CREATE TABLE|DATABASE t|db`, but `SHOW t` and `SHOW db` are not supported.

:::note
if you use this statement to get the `CREATE` query of system tables,
you will get a *fake* query, which only declares the table structure,
but cannot be used to create a table.
:::

## SHOW DATABASES

This statement prints a list of all databases.

### Syntax

```sql title="Syntax"
SHOW DATABASES [[NOT] LIKE | ILIKE '<pattern>'] [LIMIT <N>] [INTO OUTFILE filename] [FORMAT format]
```

It is identical to the query:

```sql
SELECT name FROM system.databases [WHERE name [NOT] LIKE | ILIKE '<pattern>'] [LIMIT <N>] [INTO OUTFILE filename] [FORMAT format]
```

### Examples

In this example we use `SHOW` to obtain database names containing the symbol sequence 'de' in their names:

```sql title="Query"
SHOW DATABASES LIKE '%de%'
```

```text title="Response"
┌─name────┐
│ default │
└─────────┘
```

We can also do so in a case-insensitive manner:

```sql title="Query"
SHOW DATABASES ILIKE '%DE%'
```

```text title="Response"
┌─name────┐
│ default │
└─────────┘
```

> Or to obtain database names which do not contain the symbols sequence 'de' in their names:

Reviewer: `nit`
```suggestion
Or obtain database names which do not contain the symbols sequence 'de' in their names:
```

└────────────────────────────────┘
```

> Finally, we'll get the names of only the first two databases:

Reviewer: ```suggestion
Finally, we can get the names of only the first two databases:
```

```

If the `FROM` clause is not specified, the query returns a list of tables from the current database.

This statement is identical to the query:

```sql
SELECT name FROM system.tables [WHERE name [NOT] LIKE | ILIKE '<pattern>'] [LIMIT <N>] [INTO OUTFILE <filename>] [FORMAT <format>]
```

### Examples

In this example we use the `SHOW TABLES` statement to find all tables containing 'user' in their names:

```sql title="Query"
SHOW TABLES FROM system LIKE '%user%'
```

```text title="Response"
┌─name─────────────┐
│ user_directories │
│ users            │
└──────────────────┘
```

We can also do so in a case-insensitive manner:

```sql title="Query"
SHOW TABLES FROM system ILIKE '%USER%'
```

```text title="Response"
┌─name─────────────┐
│ user_directories │
│ users            │
└──────────────────┘
```

Or to find tables which don't contain the letter 's' in their names:

```sql title="Query"
SHOW TABLES FROM system NOT LIKE '%s%'
```

```text title="Response"
┌─name─────────┐
│ metric_log   │
│ metric_log_0 │
│ metric_log_1 │
└──────────────┘
```

> Finally, we'll get the names of only the first two tables:

Reviewer: ```suggestion
Finally, we can get the names of only the first two tables:
```

```

The `SELECT * FROM system.processes` query returns data about all the current queries.

:::tip
Execute in the console:

```bash
$ watch -n1 "clickhouse-client --query='SHOW PROCESSLIST'"
```
:::

## SHOW GRANTS

The `SHOW GRANTS` statement shows privileges for a user.

### Syntax

```sql title="Syntax"
SHOW GRANTS [FOR user1 [, user2 ...]] [WITH IMPLICIT] [FINAL]
```

If the user is not specified, the query returns privileges for the current user.

The `WITH IMPLICIT` modifier allows showing the implicit grants (e.g., `GRANT SELECT ON system.one`)

The `FINAL` modifier merges all grants from the user and its granted roles (with inheritance)

## SHOW CREATE USER

The `SHOW CREATE USER` statement shows parameters which were used at [user creation](../../sql-reference/statements/create/user.md).

### Syntax

```sql title="Syntax"
SHOW CREATE USER [name1 [, name2 ...] | CURRENT_USER]
```

## SHOW CREATE ROLE

The `SHOW CREATE ROLE` statement shows parameters which were used at [role creation](../../sql-reference/statements/create/role.md).

### Syntax

```sql title="Syntax"
SHOW CREATE ROLE name1 [, name2 ...]
```

## SHOW CREATE ROW POLICY

> The `SHOW CREATE ROW POLICY` statement shows parameters which used at [row policy creation](../../sql-reference/statements/create/row-policy.md).

Reviewer: ```suggestion
The `SHOW CREATE ROW POLICY` statement shows parameters which were used at [row policy creation](../../sql-reference/statements/create/row-policy.md).
```

### Syntax

```sql title="Syntax"
SHOW CREATE [ROW] POLICY name ON [database1.]table1 [, [database2.]table2 ...]
```

## SHOW CREATE QUOTA

> The `SHOW CREATE QUOTA` statement shows parameters which used at [quota creation](../../sql-reference/statements/create/quota.md).

Reviewer: ```suggestion
The `SHOW CREATE QUOTA` statement shows parameters which were used at [quota creation](../../sql-reference/statements/create/quota.md).
```

### Syntax

```sql title="Syntax"
SHOW CREATE QUOTA [name1 [, name2 ...] | CURRENT]
```

## SHOW CREATE SETTINGS PROFILE

> The `SHOW CREATE SETTINGS PROFILE` statement shows parameters which used at [settings profile creation](../../sql-reference/statements/create/settings-profile.md).

Reviewer: ```suggestion
The `SHOW CREATE SETTINGS PROFILE` statement shows parameters which were used at [settings profile creation](../../sql-reference/statements/create/settings-profile.md).
```

### Syntax

```sql title="Syntax"
SHOW QUOTAS
```

## SHOW QUOTA

The `SHOW QUOTA` statement returns a [quota](../../operations/quotas.md) consumption for all users or for current user. 
To view other parameters, see system tables [`system.quotas_usage`](../../operations/system-tables/quotas_usage.md#system_tables-quotas_usage) and [`system.quota_usage`](../../operations/system-tables/quota_usage.md#system_tables-quota_usage).

### Syntax

```sql title="Syntax"
SHOW [CURRENT] QUOTA
```
## SHOW ACCESS

The `SHOW ACCESS` statement shows all [users](../../guides/sre/user-management/index.md#user-account-management), [roles](../../guides/sre/user-management/index.md#role-management), [profiles](../../guides/sre/user-management/index.md#settings-profiles-management), etc. and all their [grants](../../sql-reference/statements/grant.md#privileges).

### Syntax

```sql title="Syntax"
SHOW ACCESS
```

## SHOW CLUSTER(S)

The `SHOW CLUSTER(S)` statement returns a list of clusters. 
All available clusters are listed in the [`system.clusters`](../../operations/system-tables/clusters.md) table.

:::note

> The `SHOW CLUSTER name` query displays the contents of `system.clusters` table for the current cluster.

Reviewer: This sounds more accurate to me:
```suggestion
The `SHOW CLUSTER name` query displays the contents of `system.clusters` table for the cluster `name`.
```
